main.go

```diff
@@ -507,6 +511,7 @@ func main() {
 	port := flag.String("port", "/dev/ttyACM0", "flash port")
 	cFlags := flag.String("cflags", "", "additional cflags for compiler")
 	ldFlags := flag.String("ldflags", "", "additional ldflags for linker")
+	wasmI64Enable := flag.Bool("wasmi64enable", false, "enable i64 in WebAssembly exports")
```

Comment:
Not sure about this name, and the other variable names as well.
`i64` is always enabled in the WebAssembly backend. There is no problem at all with passing it between wasm-to-wasm functions. Only external JS calls (crossing the JS-WASM barrier) cannot deal with `i64`. This option seems to indicate that without it, `i64` is not enabled.
What about `-wasm-cc` or `-wasm-abi` (string)? You can set it to "" or "js", defaulting to "js" at the moment, for the calling convention used on ABI boundaries. An empty string does the obvious thing, which is emitting plain `i64` arguments and returns. More calling conventions may be added in the future.